contracts/contracts/ERC677/ERC20/ERC20_base.cairo

```diff
@@ -0,0 +1,192 @@
+%lang starknet
```

Comment:
Is this a fork (any change? - see how hard is to assess this by just looking at the code) or a duplicate of the vendor contract?

If this is a duplicate, please put it into the vendor folder, recreate the path as at source, like we did for bridge vendor contracts while building L2EP.

Reply:
Done, vendor create